.github/workflows/main.yml

```diff
@@ -39,10 +39,11 @@ jobs:
         pip install -r ./collect_metadata/requirements.txt
         pip install -r ./short_term/requirements.txt
         pip install -r ./long_term/requirements.txt
+        pip install -r ./dashboard/requirements.txt
```

Comment:
I feel there must be an easier way to do this other than installing each file by hand...